lib/mayaUsd/utils/utilSerialization.h

```diff
@@ -56,18 +56,39 @@ enum USDUnsavedEditsOption
     kIgnoreUSDEdits
 };
 /*! \brief Queries the Maya optionVar that decides which saving option Maya
-    shoudl use for Usd edits.
+    should use for Usd edits.
  */
 MAYAUSD_CORE_PUBLIC
 USDUnsavedEditsOption serializeUsdEditsLocationOption();
 
+/*! \brief Utility function to update the file path attribute on the proxy shape
+    when an anonymous root layer gets exported to disk.
+ */
+MAYAUSD_CORE_PUBLIC
+void setNewProxyPath(const MString& proxyNodeName, const MString& newValue);
+
+struct LayerParent
+{
+    // Every layer that we are saving should have either a parent layer that
+    // we will need to remap to point to the new path, or the stage if it is an
+    // anonymous root layer.
+    SdfLayerRefPtr _layerParent;
+    UsdStageRefPtr _stageParent;
+};
+
+struct stageLayersToSave
+{
+    std::vector<std::pair<SdfLayerRefPtr, LayerParent>> _anonLayers;
+    std::vector<SdfLayerRefPtr>                         _dirtyFileBackedLayers;
+};
```

Comment:
Do these two structs need to have the "MAYAUSD_CORE_PUBLIC" so they will be exported as well?